composer.json

```diff
@@ -16,9 +16,9 @@
         "psr-4": {"OckCyp\\CoversValidator\\Tests\\": "tests/"}
     },
     "require": {
-        "php": "^7.0 || ^8.0",
-        "phpunit/phpunit": "^6.0 || ^7.0 || ^8.0 || ^9.0",
-        "symfony/console": "^2.7 || ^3.0 || ^4.0 || ^5.0 || ^6.0"
+        "php": ">=7.1",
```

Comment:
is there a reason to not drop even lower the versions? 7.4 is going EOL soon so you could put it as minimum and still be considered very lenient in terms of support

Reply:
I couldn't find any compatibility code necessary for PHP versions between 7.1 and 7.4. PHP 8.0 will bring the "mixed" type, so I guess we can drop more versions when PHP 8.0 is as popular as PHP 7.4 is now